kv_store/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "kv_store"
+description = "A KV Store based on edfsm FSM"
+readme = "README.md"
+
+version.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+
+[dependencies]
+edfsm.workspace = true
+machine = { workspace = true, optional = true }
+smol_str = { version = "0.3.2", default-features = false, features = ["serde"] }
```

Comment:
`version = "0.3.2"` can be replaced with `workspace = true` with `smol_str` declared there.